packages/components/src/mobile/keyboard-aware-flat-list/index.android.js

```diff
@@ -2,17 +2,18 @@
  * External dependencies
  */
 import { FlatList } from 'react-native';
-
```

Comment:
Not sure if this line being removed was intentional, the codebase seems to use an empty line in these situations (it would be nice though to have a linter rule to ensure consistency, but that's another topic).

Reply:
Thanks for spotting. I will remove it.